spec/initializers/logstop_spec.rb

```diff
@@ -0,0 +1,152 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'logstop'
+
+# rubocop:disable RSpec/DescribeClass
+RSpec.describe 'Logstop PII filtering' do
+  let(:va_custom_scrubber) do
+    lambda do |msg|
```

Comment:
thought: maybe we could somehow test the lambda actually used in `config/initializers/logstop.rb` here instead of crafting a distinct one? (suggest maybe wrapping the lambda def there in a module and defining it as a static/method on `self.` that returns the lambda so we could access here)